.github/workflows/android.yml

```diff
@@ -19,21 +19,17 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt install libasound2-dev libavcodec-dev libavformat-dev libavutil-dev libgl-dev libgtk-3-dev libpango1.0-dev libxtst-dev
-
-      - name: Setup Android Keystore
-        id: android_keystore_file
-        uses: timheuer/base64-to-file@v1
-        with:
-          fileName: 'my.keystore'
-          encodedString: ${{ secrets.GLUON_ANDROID_KEYSTORE_BASE64 }}
-
+            
       - name: Gluon License
         uses: gluonhq/gluon-build-license@v1
         with:
           gluon-license: ${{ secrets.GLUON_LICENSE }}
 
       - name: Gluon Build
-        run: mvn -Pandroid gluonfx:build gluonfx:package
+        run: |
+          export ANDROID_SDK=$ANDROID_HOME
+          export ANDROID_NDK=$ANDROID_HOME/ndk/23.2.8568313
```

Comment:
What are the chances of this version changing in the future?

Reply:
Since we use ubuntu-latest, that is likely to happen, as soon as they move from 20.04 to 22 or so, possibly removing the older NDK in the process... But once that happen we'll notice and react accordingly.

Reply:
Should we hardcode the latest version for now to avoid failing the build in future?

Reply:
The latest version (25) doesn't work, that's why I'm using the older possible.

Reply:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2204-Readme.md#android has 23 as well